### PR TITLE
Loaders: material.shading ==> material.flatShading

### PR DESCRIPTION
**Pull request by WestLangley**

### CODE REVIEW

File: examples/js/loaders/WWOBJLoader2.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-
-'use strict';
-
+/**
+  * @author Kai Salmen / https://kaisalmen.de
+  * Development repository: https://github.com/kaisalmen/WWOBJLoader
+  */
+
+'use strict';
+
```
Comment (WestLangley) [inline examples/js/loaders/WWOBJLoader2.js]: Not sure what causes these header diffs...

**Comment by WestLangley** [comment]
@Mugen87 I have omitted `ColladerLoader` because I did not know what to make of "preferrred shading". What do you recommend?

File: examples/js/loaders/WWOBJLoader2.js (hunk 1 above)
Comment (Mugen87) [inline examples/js/loaders/WWOBJLoader2.js]: 🤔

**Comment by Mugen87** [comment]
IMO, we can remove `preferredShading` from `ColladaLoader`. It looks like that the Collada spec does not define any material/effect properties that control smooth or flat shading. I think it's better when loaders only focus on the respective format and do not provide additional features or functions like a `setPreferredShading` interface.

**Comment by mrdoob** [comment]
+1

**Comment by WestLangley** [comment]
@Mugen87 @mrdoob Done. #11735.

This PR should be good-to-go.

**Pull request merged**